cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -190,6 +190,7 @@ func (dc *DownstreamController) addToConfigMap(device *v1alpha2.Device) {
 				klog.Errorf("Failed to create config map %v in namespace %v, error %v", nodeConfigMap, device.Namespace, err)
 				return
 			}
+			return
```

Comment:
First we should use the `apierrors.IsNotFound` to check for `err`
```go
import apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Reply:
+1, we should check whether the err is NotFound type, if not, we should log err and return